security/address-bar-spoofing/spoof-js-download-url.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width">
+  <title>URL Spoofing - Download URL</title>
+  <script>
+    // eslint-disable-next-line no-unused-vars
+    function run() {
+      const w = open()
+      w.opener = null
+      w.document.write('<h1>Not Google.</h1>')
+      w.location = 'http://goo.gl/3Hywh'
```

Comment:
not sure if google / shortcut is needed here, but we could avoid depending on google by using one of our servers and one of existing download paths e.g https://bad.third-party.site/features/download/file/pdf